x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_add_edit/form/field_config.tsx

```diff
@@ -1251,4 +1281,168 @@ export const FIELD = (readOnly?: boolean): FieldMap => ({
       isDisabled: readOnly,
     }),
   },
+  [ConfigKey.MODE]: {
+    fieldKey: ConfigKey.MODE,
+    component: Select,
+    label: i18n.translate('xpack.synthetics.monitorConfig.mode.label', {
+      defaultMessage: 'Mode',
+    }),
+    helpText: (
+      <FormattedMessage
+        id="xpack.synthetics.monitorConfig.syntheticsArgs.mode.helpText"
+        defaultMessage="If mode is {any}, the monitor pings only one IP address for a hostname. If mode is {all}, the monitor pings all resolvable IPs for a hostname. The mode: {all} setting is useful if you are using a DNS-load balancer and want to ping every IP address for the specified hostname."
+        values={{
+          all: <EuiCode>all</EuiCode>,
+          any: <EuiCode>any</EuiCode>,
+        }}
+      />
+    ),
+    props: (): EuiSelectProps => ({
+      options: Object.values(Mode).map((value) => ({
+        value,
+        text: value,
+      })),
+      disabled: readOnly,
+    }),
+  },
+  [ConfigKey.RESPONSE_BODY_MAX_BYTES]: {
+    fieldKey: ConfigKey.RESPONSE_BODY_MAX_BYTES,
+    component: FieldNumber,
+    label: i18n.translate('xpack.synthetics.monitorConfig.responseBodyMaxBytes.label', {
+      defaultMessage: 'Response body max bytes',
+    }),
+    helpText: i18n.translate('xpack.synthetics.monitorConfig.responseBodyMaxBytes.helpText', {
+      defaultMessage: 'Control the maximum size of the stored body contents.',
```

Comment:
```suggestion
      defaultMessage: 'Controls the maximum size of the stored body contents.',
```
What happens when the response body size is above the value set?
It could be something like 
`Maximum size of the indexed response body. When the response body is above this value, the monitor fails/the body is not indexed/other.`

I'm using "indexed response body" instead of "stored body contents" because it's the name of the previous setting in the screenshot, so I assume the settings are directly related?

Feel free to correct or ignore me, I'm not sure I get this right :)